src/drivers/general/queries/ExtendedDeprecatedApis/ExtendedDeprecatedApis.ql

```diff
@@ -0,0 +1,443 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+/**
+ * @name Use of deprecated function or macro (C28719, C28726, C28750)
+ * @description Unsafe, deprectated APIs should not be used.  This is a port of Code Analysis checks C28719, C28726, and C28750.
```

Comment:
deprecated

Reply:
Good catch, thanks.  BTW, when you find a minor typo like this, GitHub has an option to provide an inline change suggestion that I can accept and merge by clicking on the "+-" button in the formatting area while writing a comment.